lib/__tests__/_processQueue.test.ts

```diff
@@ -0,0 +1,70 @@
+import { processQueue } from "../_processQueue";
+
+const makeGlobalObject = () => {
+  // this is a simplified typescript tolerable version of the code we ask our
+  // customers to embed when installing the insights client in the browser.
+  // cf. https://github.com/algolia/search-insights.js#loading-and-initializing-the-library
+  const globalObject: any = {};
+  globalObject.AlgoliaAnalyticsObject = "aa";
+  globalObject.aa = function() {
+    globalObject.aa.queue = globalObject.aa.queue || [];
+    globalObject.aa.queue.push(arguments);
+  };
+  return globalObject;
+};
+
+class AlgoliaAnalytics {
+  public init: Function;
+  public otherMethod: Function;
+  public processQueue: Function;
+  constructor() {
+    this.init = jest.fn();
+    this.otherMethod = jest.fn(() => "otherMethodReturnedValue");
+    this.processQueue = processQueue.bind(this);
```

Comment:
We could add a comment that explain that it's actually the function that we test.

Reply:
done here 8798f3e3aebc9b974b132bd430a0f4eb064be39a